buildSrc/src/main/kotlin/org.plan.research.kotlinx-fuzz-module.gradle.kts

```diff
@@ -33,3 +38,33 @@ tasks.test {
         showStackTraces = true
     }
 }
+
+tasks.register("setLoggingProperties") {
+    val logback = rootProject.file("logback.xml")
+    val backupFile = rootProject.file("logback.xml.bak")
+    doFirst {
+        if (!rootProject.extra.has("logbackModified") || rootProject.extra["logbackModified"] == 0) {
+            rootProject.extra["logbackModified"] = 0
+            logback.copyTo(backupFile, overwrite = true)
+            val envValue = System.getProperty("KOTLINX_FUZZ_LOGGING_LEVEL", "OFF")
+            val oldContent = logback.readText().split("<configuration>\n")
+            logback.writeText(
+                "${oldContent[0]}<configuration>\n<property name=\"KOTLINX_FUZZ_LOGGING_LEVEL\" value=\"$envValue\"/>\n${oldContent[1]}"
+            )
+        }
+        rootProject.extra["logbackModified"] = rootProject.extra["logbackModified"] as Int + 1
+    }
+
+    doLast {
+        if (rootProject.extra["logbackModified"] == 1) {
+            backupFile.copyTo(logback, overwrite = true)
+            backupFile.delete()
+        }
+        rootProject.extra["logbackModified"] = rootProject.extra["logbackModified"] as Int - 1
+    }
+}
```

Comment:
do I understand correctly that we want to take user's logback.xml, modify it before fuzz task is run, and revert it to original once fuzz task finishes? if yes, are we *__sure__* this is an optimal way of logging from a library?

Reply:
Yes, we are adding and removing a line from the user's logback.xml. I am not sure if this is an optimal solution, however, I couldn't come up with another solution, since conditions didn't work as expected